development/Dockerfile

```diff
@@ -42,4 +42,8 @@ COPY . /opt/nautobot
 
 RUN poetry install
 
+# Install additional requirements from local_requirements.txt
+RUN if [ -f local_requirements.txt ] ; then cat local_requirements.txt | grep -E '^[^# ]' | cut -d= -f1 | xargs -n 1 poetry add ; fi
```

Comment:
Huh, this is kind of clever, but not sure what this buys us in a container compared to just running `if [ -f local_requirements.txt ] ; then
 pip install -r local_requirements.txt; fi`.

Thoughts?

Reply:
It at least has the small advantage that it stays within the Poetry tool chain instead of mixing Poetry and Pip. I'd want to test this to make sure it doesn't end up modifying the pyproject.toml/poetry.lock in Git, though?

Reply:
@glennmatthews Yeah that was my concern, too, I think. I just didn't spell it out. Pretty sure it would update them which might result in breaking layer caching.